src/main/java/com/minecolonies/coremod/commands/citizencommands/ListCitizensCommand.java

```diff
@@ -59,37 +72,63 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
         return super.getCommandUsage(sender) + "<colonyId>";
     }
 
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final ActionMenu actionMenu) throws CommandException
+    {
+        final Colony colony = actionMenu.getColonyForArgument("colony");
+        final Integer page = actionMenu.getIntegerForArgument("page");
+        executeShared(server, sender, colony, page);
+    }
+
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
     {
         int colonyId = getIthArgument(args, 0, getColonyId(sender));
+        final Integer page = getIthArgument(args, 1, 1);
 
+        Colony colony = null;
         if (sender instanceof EntityPlayer)
         {
             if (colonyId == -1)
             {
-                final IColony colony = ColonyManager.getIColonyByOwner(sender.getEntityWorld(), (EntityPlayer) sender);
-                if (colony != null)
+                final IColony icolony = ColonyManager.getIColonyByOwner(sender.getEntityWorld(), (EntityPlayer) sender);
+                if (icolony != null)
                 {
-                    colonyId = colony.getID();
+                    colonyId = icolony.getID();
                 }
             }
+        }
+        colony = ColonyManager.getColony(colonyId);
+
+        executeShared(server, sender, colony, page);
+    }
 
+    private void executeShared(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, final Colony colony, final Integer pageProvided) throws CommandException
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "server". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)